src/components/editArrangementTitle/editArrangementTitle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Typography, TextField } from '@material-ui/core'
+
+export class EditArrangementTitle extends Component {
+    constructor (props) {
+        super(props)
+
+        this.handleKeyPress = this.handleKeyPress.bind(this)
+        this.escFunction = this.escFunction.bind(this);
+    }
+    
+    state = {
+        name: ''
+      };
+   
+    handleChange = name => event => {
+        this.setState({
+          name: event.target.value
+        });
+      };
+    
+    handleKeyPress (event) {
+        if (event.key === 'Enter') {
+            this.setState({
+                name: this.state.name
+            });
+            this.props.handleEnter(this.state.name)
+        }
+    }
+
+    escFunction (event) {
+        if(event.keyCode === 27) {
+            this.props.handleEsc()
+        }
+    }
+
+    componentDidMount(){
+        document.addEventListener("keydown", this.escFunction, false);
+    }
+
+    componentWillUnmount(){
+        document.removeEventListener("keydown", this.escFunction, false);
+    }
+
+    render () {
+		return (
+			<Typography variant="headline" align="left">
+				<TextField
+                    id="outlined-name"
```

Comment:
Nitpick: can we tab the values over so that the formatting looks lsightly better?

Reply:
agreed

Reply:
It is indented nicely in my VS Code. not sure why it looks ugly here

Reply:
probably because of tabs?

Reply:
I changed any tabs I had to spaces